src/Resources/js/modules/combined-meal-service.ts

```diff
@@ -1,4 +1,10 @@
 export class CombinedMealService {
+
+    /**
+     * No. of dish items in a combined dish.
+     */
+    private static readonly COMBINED_DISH_ITEM_COUNT = 2;
+
     public static getCombinedMealDishes($mealContainer: JQuery): Dish[] {
```

Comment:
Some refactoring suggestions: getCombinedDishes

Reply:
Shall we get rid of the "combined" word altogether? The service name speaks for itself.

Reply:
Good point!